docs/general/attributes.md

```diff
@@ -423,40 +423,6 @@ For an HTTP server span, this would be the function that handles the incoming re
 The attributes listed below allow to report this unit of code and therefore to provide more context
 about the span.
```

Comment:
the attributes are not listed below anymore, so we need to adjust the wording.

Reply:
Right, updated!